### PR TITLE
Docs improve api docs

### LINKED ISSUE #526: Improve api documentation
The api is only partially documented in the online docs, for instance the flags page lists attributes (but capitalises them) and does not list methods - it would be good to document the public methods

https://waffle.readthedocs.io/en/stable/types/flag.html

[Switches ](https://waffle.readthedocs.io/en/stable/types/switch.html)lists neither attributes nor methods

[Samples](https://waffle.readthedocs.io/en/stable/types/sample.html) lists attributes (but capitalises them) and does not list methods

### PR DESCRIPTION
**Pull request by dancergraham**
Add missing attributes and methods

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -119,7 +119,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```
Comment (clintonb) [inline docs/conf.py:122]: Why was this removed?

Comment (dancergraham) [reply docs/conf.py:122]: Hi, thanks for looking at this - this line raises a warning during build because there is nothing in there - I just commented it to make it easy to find again later if we want to add any static assets

Comment (dancergraham) [reply docs/conf.py:122]: You can see the warning on line 7 of the `Build docs` phase of my other PR, which adds tests for pull requests that touch docs

https://github.com/jazzband/django-waffle/actions/runs/12378362580/job/34550046030

**Review by clintonb — APPROVED** [review]
